README.md

```diff
@@ -1,4 +1,51 @@
-# nswag-extensions
-Nswag extension to support query string versioning
+# Ibistic.Public.NSwag.Extensions
+A framework that provides extensions to create OpenAPI 3.0 specifications with NSwag.
 
-Pending to add info to the readme file.
+NSwag toolchain used from https://github.com/RicoSuter/NSwag under the license https://github.com/RicoSuter/NSwag/blob/master/LICENSE.md.
+
+Please only use this component if you agree to and comply with the above license terms.
+
+## Install
+
+To install Ibistic.Public.NSwag.Extensions, run the following command in the Package Manager Console
+
+    PM> Install-Package Ibistic.Public.NSwag.Extensions
+
+## Documentation
+
+Ibistic.Public.NSwag.Extensions has by now one new api version processor.
+
+### ApiVersionQueryStringProcessor
+
+The api version processor from NSwag (NSwagProcessors.ApiVersionProcessor) only works when the versioning strategy used is based on the uri, i.e. /api/1.0/helloworld.
+
+ApiVersionQueryStringProcessor has been built to support query string versioning, which is the default versioning mode in Web API 2, i.e. /api/helloworld?api-version=1.0.
+
+The way to use it is just to specify the new processor when the Open API specification is gonna be created. There are several ways to create that specification, by command line would be:
```

Comment:
remove "gonna be"

Reply:
Change "by command line would be" to "e.g using command line:"

Reply:
Done.